src/main/java/core/mate/academy/service/producers/BulldozerProducer.java

```diff
@@ -0,0 +1,21 @@
+package core.mate.academy.service.producers;
+
+import core.mate.academy.model.Bulldozer;
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+    private static final int NUM_OF_MACHINES = 3;
+    private List<Bulldozer> machines = new ArrayList<>();
+
+    @Override
+    public List<Bulldozer> get() {
+        int randomPower = new Random().nextInt(100);
+        for (int i = 0; i < randomPower; i++) {
+            machines.add(new Bulldozer(randomPower));
+        }
+        return machines;
+    }
```

Comment:
this whole class just looks terrible: 
* you get a random 0-99 number (power can be 0?) 
* `100` should be a constant
* you create the same amount of bulldozers as you got this power
* unused `NUM_OF_MACHINES` constant
* you set the same power to all of those bulldozers (what is the point of random?)
* you do not reuse a `Random` instance as a class field, which we discussed around 2 months ago. 
* why do you have common `machines` list for a whole class? Do you realize that method `get()` will add more and more machines to it after each get() call?

So many and so common and silly mistakes in such a small block of code... Please think more about what you do, what your code actually does, and how does it look, 😞 
Same mistakes in other `Producer` classes